help/destinations/catalog/analytics/gainsight-px.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Gainsight PX
+description: The Gainsight PX destination allows allows segmentation information to be pushed to Gainsight PX 
+---
+
+# Gainsight PX connection {#gainsight-px}
+
+
+## Overview {#overview}
+
+[!DNL Gainsight PX](https://www.gainsight.com/product-experience/) is a product experience platform that enables product teams to understand how users use their products, collect feedback, and create in-app engagements like product walkthroughs to drive user onboarding and product adoption.
+
+>[!IMPORTANT]
+>
+>The destination connector and documentation page are created and maintained by the *Gainsight PX* team. For any inquiries or update requests, please contact them directly at *`pxsupport@gainsight.com`.*
+ 
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the *Gainsight PX* destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case #1 {#use-case-1}
+
+*Targeting In-App Engagements:*
+
+*A SaaS company wants to engage their customers via an in-application guide constructed on Gainsight PX.  An audience to receive this engagement has been built on Adobe Experience Platform and the Gainsight PX destination will consume the audience and make it available within the Gainsight PX environment.*
+
+## Prerequisites {#prerequisites}
+
+* Contact the [!DNL Gainsight] support team and request the activation of external segment features for your subscription.
+* Generate an OAuth Secret value for your PX subscription, using the Generate New Secret button at the bottom of the [Company Details page] (https://app.aptrinsic.com/settings/subscription)
+![Company Details screen in Gainsight PX showing the Generate New Secret button](../../assets/catalog/analytics/gainsight_px/generate_oauth_secret.png)
+
+## Supported identities {#supported-identities}
+
+
+*Gainsight PX* supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+| Target Identity |Description| Considerations                                                                                                                                                                                                                                                                         |
+|---|---|---|
+| IdentifyID      |Custom user IDs| Use this namespace to use whatever identify is used when identifying users within PX.                                                                                                                                                                                                  |
```

Comment:
I don't understand what this description is supposed to say. There is also a typo there (whatever identify).

Please rephrase the description in a clearer way. What does the IdentifyID do? What values does it accept? Could you provide an example of how to use it?